modules/core/test/test_intrin_utils.hpp

```diff
@@ -168,7 +44,8 @@ template <typename R> struct Data
     }
     operator R () const
     {
-        return initializer<VTraits<R>::max_nlanes>().init(*this);
+        CV_DbgAssert(VTraits<R>::vlanes() <= VTraits<R>::max_nlanes);
```

Comment:
Why do we need to limit for debug only?

Reply:
Tried to make similar to removed assert. I agree, regular `CV_Assert` would be better, I'll fix it.